tabpfn_client/client.py

```diff
@@ -117,19 +126,25 @@ def predict(self, train_set_uid: str, x_test):
 
         return np.array(response.json()["y_pred"])
 
-    def error_raising(self, response, method_name):
+    def error_raising(self, response, method_name, only_version_check=False):
         if response.status_code != 200:
             load = None
             try:
                 load = response.json()
             except Exception:
                 pass
+            if (response.status_code == 403 and load and load.get("detail").startswith("Client version too old")
+                    or response.status_code == 400 and load and load.get("detail").startswith("Client version")):
+                raise RuntimeError(load.get("detail"))
+            if only_version_check:
```

Comment:
We could use some comments in the following lines to better demonstrates the logic.